docker-compose.yml

```diff
@@ -52,6 +52,9 @@ services:
       - TUMMY_DB=tummy
       - TUMMY_PORT=${TUMMY_PORT}
       - TUMMY_HOST=tummy
+      - ZIPFILE_PATH=${ZIPFILE_PATH}
```

Comment:
Is this env var used anywhere in the code? If the zipfile is mounted at `/slack-export.zip`, this variable should not be required, right?

Reply:
Also, please update the `.env.template`.

Reply:
let's update the .env.template post discussion on makefile